frontend/src/pages/team/Settings/Devices.vue

```diff
@@ -0,0 +1,216 @@
+    watch: {
+        team: 'fetchData'
+    },
+    async mounted () {
+        await this.fetchData()
+        this.loading = false
+        this.checkInterval = setTimeout(this.pollForData, 10000)
+    },
+    unmounted () {
+        clearInterval(this.checkInterval)
+    },
+    methods: {
+        async pollForData () {
+            try {
+                await this.fetchData(null, true) // TODO: Implement pagination
+            } finally {
+                this.checkInterval = setTimeout(this.pollForData, 10000)
+            }
+        },
+        async fetchData (nextCursor = null, polled = false) {
+            // load project names into local cache (TODO: consider moving this to a vuex store for global access)
+            this.projectNames = await teamApi.getTeamProjectList(this.team.id)
+            // get the tokens
+            const data = await teamApi.getTeamDeviceProvisioningTokens(this.team.id, nextCursor)
+
+            // Polling never resets the devices list
+            if (!nextCursor && !polled) {
+                this.tokens = new Map()
+            }
+            data.tokens.forEach(token => {
+                this.updateTokenCache(token)
+            })
+
+            // TODO: Implement pagination
+            if (!polled) {
+                this.nextCursor = data.meta.next_cursor
+            }
+        },
+        getProjectName (id) {
+            const project = this.projectNames?.find(p => p.id === id)
+            return project ? project.name : id
+        },
+        async loadMore () {
+            await this.fetchData(this.nextCursor)
+        },
+        showCreateDeviceDialog () {
+            this.$refs.CreateProvisioningTokenDialog.show(null, this.project)
+        },
+        showEditDialog (token) {
+            this.$refs.CreateProvisioningTokenDialog.show(token)
+        },
+        tokenCreating () {
+            this.creatingDevice = true
+        },
+        async tokenCreated (token) {
+            this.creatingDevice = false
+            if (token) {
+                setTimeout(() => {
+                    this.$refs.provisioningCredentialsDialog.show(token)
+                }, 500)
+                this.updateTokenCache(token)
+            }
+        },
+        tokenUpdated (token) {
+            this.updateTokenCache(token)
+        },
+        updateTokenCache (token) {
+            token.projectName = this.getProjectName(token.project)
+            this.tokens.set(token.id, token)
+        },
+        menuAction (action, tokenId) {
+            const token = this.tokens.get(tokenId)
+            if (action === 'edit') {
+                this.showEditDialog(token)
+            } else if (action === 'delete') {
+                Dialog.show({
+                    header: 'Delete Provisioning Token',
+                    kind: 'danger',
+                    text: 'Are you sure you want to delete this provisioning token? Once deleted, devices with this token will no longer be auto provisioned.',
```

Comment:
```suggestion
                    text: 'Are you sure you want to delete this provisioning token? Once deleted, it can no longer be used to provision new devices to the team.',
```

I'm concerned we may give the impression that this will cut off all devices that ever used this token to get registered - rather than just new ones.

Reply:
Addressed in [3040b7a](https://github.com/flowforge/flowforge/pull/1664/commits/3040b7ab585249b218118051d47662bac0ebbaad)

Reply:
Not seeing any changes to this line of text in the branch.